tests/test_virtualenv.py

```diff
@@ -595,3 +595,14 @@ def test_create_environment_with_old_pip(tmpdir):
         shutil.copy(str(old_dep), support_dir)
     venvdir = str(tmpdir / "venv")
     virtualenv.create_environment(venvdir, search_dirs=[support_dir], no_wheel=True)
+
+
+def test_license_builtin(clean_python):
+    _, bin_dir, _ = clean_python
+    proc = subprocess.Popen(
+        (os.path.join(bin_dir, "python"), "-c", "license()"), stdin=subprocess.PIPE, stdout=subprocess.PIPE
```

Comment:
probably want universal newlines here to avoid issues with Windows

Reply:
the windows issue is actually a more subtle one.  Since the `tox` venv is created with old virtualenv (and copies) -- there's no `LICENSE` file there, so when it goes to copy from there, there's no license.

I'll have to put some more thought into this 🤔